ext/pdo/pdo_stmt.c

```diff
@@ -764,7 +764,7 @@ static int make_callable_ex(pdo_stmt_t *stmt, zval *callable, zend_fcall_info *
 		return 0;
 	}
 	if (is_callable_error) {
-		/* Possible E_STRICT error message */
+		/* Possible error message */
 		efree(is_callable_error);
 	}
```

Comment:
Is this maybe related to #4406 with the E_STRICT message generation?
I don't know how PDO works and if it calls the underlying "normal" driver (which for mysql would be mysqlnd).
Should this check maybe be dropped entirely ?

Reply:
This is related to deprecation warning for calling non-static methods statically, I believe.

Reply:
Okay so leaving it as if.